### PR TITLE
Various typo fixes, clarifications and additions to the current Rev News

### PR DESCRIPTION
**Pull request by dscho**

### CODE REVIEW

File: rev_news/drafts/edition-24.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ __Events__
 
 ___Git Merge 2017___
 
-Apparently, an increasing number of excellent bloggers attented the conference:
+Apparently, an increasing number of excellent bloggers attend the conference:
```
Comment (chriscool) [inline rev_news/drafts/edition-24.md]: I think it should be "attended".

File: rev_news/drafts/edition-24.md [hunk 2]
```diff
@@ -185,12 +185,13 @@ the [Git Merge 2017](http://git-merge.com/) conference that happenend on Februar
...
-                  in the email. At present, the “what’s cooking” email
-                  is mostly generated in an automated way from the
-                  [`todo` branch](https://github.com/git/git/commits/todo)
-                  using a special-purpose shell scripts.
+		* The [“What’s cooking” emails](https://github.com/git/git/blob/v2.11.1/Documentation/SubmittingPatches#L384-L386)
+                  are sent only to the mailing list, but none of the original
+                  contributors are Cc:ed in that email. At present, the
+                  “what’s cooking” email is mostly generated in an automated
+                  way from the [`todo`
+                  branch](https://github.com/git/git/commits/todo) using
```
Comment (chriscool) [inline rev_news/drafts/edition-24.md]: I am always afraid by links that span several lines as I had problems with them before.

File: rev_news/drafts/edition-24.md [hunk 3]
```diff
@@ -224,6 +224,7 @@ the [Git Merge 2017](http://git-merge.com/) conference that happenend on Februar
 	appeared in the Linux Kernel world (especially on the Intel side).
 	Johannes emphazised his willingness to establish tools which help to improve the situation.
 	It was mentioned that code/commit notes should point to mailing list where appropriate.
+        Josh Triplett steered the discussion toward distributed review tools, and mentioned that there is already a data format for that purpose: ReviewDB. This format, however, would need to be extended to allow for addressing the issues raised in the discussion.
```
Comment (chriscool) [inline rev_news/drafts/edition-24.md]: Nit: it would be nice if this was split into several lines and indented.

**Comment by chriscool** [comment]
Ok I have a few minor comments, but as overall it looks very good, let's merge this for now.

**Comment by chriscool** [comment]
Thanks a lot!

**Comment by dscho** [comment]
> I think it should be "attended".

If you wrote "Apparently, an increasing number of excellent bloggers attend the conference" then it really means that during *this* conference, more and more bloggers arrived. That is not what we meant here, right? If you write "an increasing number of bloggers attend the conference", then you are referring to the conference as a recurring event.

**Comment by chriscool** [comment]
Yeah, I understand. The sentence we are discussing was written by @mjaix, and I am just guessing that he wanted to write "attended", but maybe it's better with "attend" anyway.

**Comment by mjaix** [comment]
IMHO "attend" sounds better, and was the intention :-) ... thanks!

**Pull request merged**